src/main/kotlin/coverosR3z/persistence/PureMemoryDatabase.kt

```diff
@@ -48,7 +48,7 @@ class PureMemoryDatabase {
 
     fun addNewUser(userName: UserName, hash: Hash, salt: String) : Int {
         val newIndex = users.size + 1
-        users.add(User(newIndex, userName.value, hash, salt))
+        users.add(User(newIndex, userName.value, hash, salt, null))
```

Comment:
Is this right? null for the employee id?

Reply:
At the moment, it seems that EmployeeID is not implemented. We will keep this in mind for proper implementation going forward, but will leave it in at this time.